JeMPI_Apps/JeMPI_API/src/main/java/org/jembi/jempi/api/HttpServer.java

```diff
@@ -411,6 +412,8 @@ private Route createRoute(final ActorSystem<Void> actorSystem, final ActorRef<Ba
                                         path("Link",
                                                 () -> routeLink(actorSystem, backEnd)))),
                                 get(() -> concat(
+                                        path("fields",
```

Comment:
I wonder if the path should just be `config` since we don't know what this might be used for in the future. There may be things other than fields in there eventually.

Reply:
E.g. maybe we eventually want to display the matching rules in the UI.

Reply:
@marrouchi are you happy for me to make this change?

Reply:
@rcrichton are you happy with the rest of the changes?

Reply:
I'm happy.

Reply:
@MatthewErispe @rcrichton I just applied the change :)